api/lib/opentelemetry/internal.rb

```diff
@@ -20,5 +20,17 @@ def printable_ascii?(string)
     def boolean?(value)
       value.is_a?(TrueClass) || value.is_a?(FalseClass)
     end
+
+    def valid_key?(key)
+      key.instance_of?(String)
+    end
+
+    def valid_value?(value)
+      value.instance_of?(String) || value == false || value == true || value.is_a?(Numeric)
+    end
+
+    def valid_attributes?(attrs)
```

Comment:
I moved `valid_attributes?` and friends to the `Internal` module. I'm not sure that this is the best place for these methods, or if they should keep the same names if it is. We could alternatively extract this concern into its own module if it makes sense.

Reply:
👍 for leaving it here for now.